docker/nginx/default.conf

```diff
@@ -22,18 +22,36 @@ server {
     send_timeout                300;
 
     location /static/ {
+        if ($http_x_forwarded_proto = "http") {
+            return 301 https://$host$request_uri;
+        }
+        if ($http_x_forwarded_proto = "https") {
+            add_header Strict-Transport-Security "max-age=3600;" always;
```

Comment:
That article is scary. I think we can assume that of the protocol is not http then it will be https, so we don't need the if at all.